public/app.js

```diff
@@ -19,7 +19,8 @@ $(function() {
             method: 'POST',
             dataType: 'json',
             data: {
-                phoneNumber: $('#phoneNumber').val()
+                phoneNumber: $('#phoneNumber').val(),
+                salesNumber: $('#salesNumber').val()
```

Comment:
What do you think about change:
```javascript
$.ajax({
        url: '/call',
        method: 'POST',
        dataType: 'json',
        data: {
            phoneNumber: $('#phoneNumber').val(),
            salesNumber: $('#salesNumber').val()
        }
}
```
Into:
```javascript
$.post(
        url: '/call',
        dataType: 'json',
        success: null,
        data: {
            phoneNumber: $.trim($('#phoneNumber').val()),
            salesNumber: $.trim($('#salesNumber').val())
        }
)
```
As in:
https://api.jquery.com/jQuery.post/#jQuery-post-url-data-success-dataType
https://api.jquery.com/jQuery.trim/#jQuery-trim-str